### PR TITLE
Global: Type ramp and link color styles for DS6 #7

### PR DESCRIPTION
**Pull request by ianmcburnie**
This simple change just adds the global anchor styles to global module for visited and unvisited links (the type ramp stuff was added in a previous PR).

I also added a "global" section to the docs to show the links and added some info around the Market Sans font usage. I added some additional helper classes for the docs page.

And yes, the docs page is still looking rudimentary at best (see #11).

screen shot 2017-09-13 at 12 10 47 pm

### CODE REVIEW

File: docs/src/less/ds6/index.less [hunk 1]
```diff
@@ -2,6 +2,10 @@
     display: none;
 }
 
+body {
+    background-color: #f8f8f8;
```
Comment (agyanchand) [inline docs/src/less/ds6/index.less]: Should we use variables to reference this color?

File: docs/src/less/ds6/index.less [hunk 2]
```diff
@@ -14,6 +18,15 @@ h3 {
     .ds6-typography-headline-3;
 }
 
+.demo {
+    background-color: white;
+    border: 1px solid #eee;
```
Comment (agyanchand) [inline docs/src/less/ds6/index.less]: Same as above for #eee and white? Shall we use variables for consistency?

File: docs/src/less/ds6/index.less (hunk 1 above)
Comment (ianmcburnie) [reply docs/src/less/ds6/index.less]: Yeah, I think we will point to the new background color variable when it's added. See #4

File: docs/src/less/ds6/index.less (hunk 2 above)
Comment (ianmcburnie) [reply docs/src/less/ds6/index.less]: I don't think these colours are in the DSL palette. Although white probably should be. Remember also this is just the CSS for the docs page, these aren't the skin src files :-)

**Comment by ianmcburnie** [comment]
Thank you @agyanchand

**Pull request merged**